ui/lib/pki/addon/routes/overview.js

```diff
@@ -1,3 +1,42 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import RSVP from 'rsvp';
 
-export default class PkiOverviewRoute extends Route {}
+export default class PkiOverviewRoute extends Route {
+  @service secretMountPath;
+  @service auth;
+  @service store;
+
+  get win() {
+    return this.window || window;
+  }
+
+  hasConfig() {
+    const endpoint = `${this.win.origin}/v1/${this.secretMountPath.currentPath}/issuers?list=true`;
+    return this.auth
+      .ajax(endpoint, 'GET', {})
+      .then(() => true)
+      .catch(() => false);
+  }
+
+  async model() {
+    return RSVP.hash({
```

Comment:
We might be able to destructure this from our import statement

import { hash } from 'rsvp';